src/clj/nnts2/handler/organization.clj

```diff
@@ -2,13 +2,37 @@
   (:require [nnts2.db.organization :as db]
             [nnts2.spec-helpers :as spec-helper]
             [nnts2.model.organization-spec :as spec]
-            [ring.util.response :as res]))
+            [nnts2.model.member-spec :as member-spec]
+            [ring.util.response :as res])
+  (:import (java.util UUID)))
 
-(defn create [{:keys [body] :as request}]
+(defn create
+  "Create an organization and make the creator an admin"
+  [{:keys [body nnts-user] :as request}]
   (if (spec/valid? body)
-    (let [org (db/insert body)]
+    (let [org (db/create body)
+          member {:user-id nnts-user
+                  :org-id  (:id org)
+                  :role    "admin"}]
       (if (nil? org)
-        (->  (res/response "Slug exists")
-             (res/status 409))
-        (res/response org)))
+        (-> (res/response "Slug exists")
+            (res/status 409))
+        (do (db/add-user member)
+            (res/response org))))
     (spec-helper/invalid (spec/explain-str body))))
+
+(defn add-user [{:keys [params]}]
+  (let [params {:user-id (UUID/fromString (:user-id params))
```

Comment:
Is this conversion to uuid necessary? You are calling db/add-user without this in the above function